### PR TITLE
[Feature] Failed proposal keep state

### PR DESCRIPTION
**Pull request by JulesGuesnon**
# Description
This PR allows to keep a proposal state if the proposal creation fails. This way, the user can update what needs to be updated and submit another proposal

### CODE REVIEW

File: components/transferForm.tsx [hunk 1]
```diff
@@ -308,6 +322,9 @@ function ExecuteContractForm(
           setField={(lambda: string, metadata: string) => {
             props.setField(lambda, metadata);
           }}
+          // onChange={shape => {
```
Comment (rueshyna) [inline components/transferForm.tsx:322]: remove it?

File: components/Sidebar.tsx [hunk 2]
```diff
@@ -169,7 +169,7 @@ const Sidebar = ({
   return (
     <aside
       className={`fixed left-0 bottom-0 ${
-        state.hasBanner ? "top-32" : "top-20"
+        state.hasBanner ? "top-32" : "top-16"
```
Comment (rueshyna) [inline components/Sidebar.tsx:172]: I think `top-20` looks nicer

this is `top-16`:
image

**Review by rueshyna — COMMENTED** [review]
The others look good to me :100:

File: components/Sidebar.tsx (hunk 2 above)
Comment (JulesGuesnon) [reply components/Sidebar.tsx:172]: What? That's quite weird, on my screen there was a gap, that's why I had to reduce it, I'll double check

Comment (JulesGuesnon) [reply components/Sidebar.tsx]: Not sure what happened, I just reversed the change and it was ok

**Pull request merged**